cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -613,12 +615,11 @@ bundle agent cfe_internal_refresh_inventory_view
       "$(sys.workdir)/httpd/php/bin/php"
         args => "$(cfe_internal_hub_vars.public_docroot)/index.php cli_tasks inventory_refresh",
         contain => silent,
-@if minimum_version(3.15.0)
```

Comment:
Since 3.15 was not supported at the (future) time of 3.24.0 release, we don't need this macro in master.